logs/2021-04-26_devOps/beautified-schedule.txt

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
These files shouldn't be included in the final thing but they're useful for seeing the sort of output I'vemanaged